src/main/java/com/epam/reportportal/spock/ReportPortalSpockListener.java

```diff
@@ -303,6 +305,13 @@ protected FinishTestItemRQ buildFinishTestItemRq(@Nonnull Maybe<String> itemId,
 		FinishTestItemRQ rq = new FinishTestItemRQ();
 		ofNullable(status).ifPresent(s -> rq.setStatus(s.name()));
 		rq.setEndTime(Calendar.getInstance().getTime());
+		if (!exceptionMessageQueue.isEmpty()) {
+			if (StringUtils.isNotEmpty(rq.getDescription())) {
```

Comment:
That's not going to work this way, `rq.getDescription()` will be always null. You have to either save the description on the item start event, either generate it here anew.

Reply:
Do you mean, like this?

```
		if (!exceptionMessageQueue.isEmpty()) {
			rq.setDescription(String.format("\nError:\n%s", exceptionMessageQueue.poll()));
		}
```

Reply:
Or do you mean to take description from StartTestItemRQ object and add to it error message?

Reply:
Second.

Reply:
Done